src/components/a-form-input/index.scss

```diff
@@ -0,0 +1,104 @@
+@import "../../styles/settings/colors";
+@import "../../styles/settings/base";
+@import "../../styles/mixins/typo";
+@import "../../styles/mixins/control-transition";
+
+.a-form-input-wrapper {
```

Comment:
@LucaMele 
@TheDadi 
I'm curious how we want to deal with class naming for the root element of a web component.
Having two blocks within the same sass file seems kinda wrong to to me.
On the other side some class like `.a-form-input__root` indicates an element by BEM syntax.
Or we could even use the overline character to indicate a parent (not a child like by `__`): `.a-form-input‾‾root` (okay this looks kinda ugly).

So I guess, we should just conclude on a naming convention - a separte block is also fine with me.